Apps/WebClient/src/NewClientApp/src/components/services/OrganDonorDetailsCard.vue

```diff
@@ -0,0 +1,170 @@
+                registrationDataValue.registrationFileId
+            )
+            .then(
+                (patientFile: PatientDataFile) =>
+                    new Blob([new Uint8Array(patientFile.content)], {
+                        type: patientFile.contentType,
+                    })
+            )
+            .then((blob) => saveAs(blob, `Organ_Donor_Registration.pdf`))
+            .catch((err) => logger.error(err));
+    }
+}
+
+function showConfirmationModal(): void {
+    sensitiveDocumentModal.value?.showModal();
+}
+</script>
+
+<template>
+    <HgCardComponent
+        title="Organ Donor Registration"
+        data-testid="organ-donor-registration-card"
+    >
+        <template #icon>
+            <img
+                src="@/assets/images/services/odr-logo.svg"
+                alt="Organ Donor Registry Logo"
+                :height="30"
+            />
+        </template>
+        <div class="flex-grow-1 d-flex flex-column text-body-1 card-content">
+            <div>
+                <span class="text-grey-darken-2">Status: </span>
+                <strong data-testid="organ-donor-registration-status">{{
+                    registrationData?.status
+                }}</strong>
+                <v-tooltip
+                    data-testid="organ-donor-registration-status-info-popover"
+                    location="top start"
+                    content-class="bg-grey-darken-3"
+                    :text="registrationData?.statusMessage"
+                >
+                    <template v-slot:activator="{ props }">
+                        <v-icon
+                            ata-testid="organ-donor-registration-status-info-button"
+                            color="primary"
+                            icon="info-circle"
+                            v-bind="props"
+                            size="x-small"
+                            class="ml-1"
+                        />
+                    </template>
+                </v-tooltip>
+            </div>
+            <div>
+                <span class="text-grey-darken-2">Decision: </span>
+                <HgButtonComponent
+                    v-if="registrationData?.registrationFileId"
+                    data-testid="organ-donor-registration-download-button"
+                    variant="secondary"
+                    :disabled="isLoadingFile"
+                    @click="showConfirmationModal"
+                >
+                    <v-progress-circular
+                        indeterminate
+                        v-if="isLoadingFile"
+                        size="20"
+                    />
+                    <v-icon
+                        v-else
+                        icon="download"
+                        size="medium"
+                        square
+                        aria-hidden="true"
+                    />
+                    <span class="text-body-1 ml-3">Download</span>
+                </HgButtonComponent>
```

Comment:
Vuetify has a loading prop available on the button component we could use instead.

```suggestion
                <HgButtonComponent
                    v-if="registrationData?.registrationFileId"
                    data-testid="organ-donor-registration-download-button"
                    variant="secondary"
                    :prepend-icon="download"
                    :text="Download"
                    :loading="isLoadingFile"
                    @click="showConfirmationModal"
                />
```